package.json

```diff
@@ -46,6 +46,11 @@
     "tsx": "^4.0.0",
     "wait-port": "^1.1.0"
   },
+  "pnpm": {
+    "overrides": {
+      "nx>axios": "^1.12.0"
```

Comment:
this was needed because wiz found a vulnerability in `axios` < `1.12.0` and the `axios` dependency is brought in through `nx`. The latest version of `nx` was still on an impacted version of `axios` so I added this override in order to force `nx` to use a later version.